### ISSUE TITLE
Keyboard layout reverts back to "us" after display power cycling

### ISSUE THREAD

**Issue by vincele** [bbidulock/icewm#698]
I have the following in ~/.icewm/preferences:
```
KeyboardLayouts = "fr"
```
If I power my display off, and then on again, the keyboard layout goes back to "us", I have to use
```
setxkbmap -layout fr
```
To get it back to french...

Am I missing something ?

icewm 3.3.0 on void linux x86_64-musl (radeon rx 550)

**Comment by gijsbers** [comment]
You want to let icewm enforce a single keyboard layout. That would be a new feature. This change allows you to click on the flag to restore your french layout.

**Comment by vincele** [comment]
Yes I want a single keyboard layout, but this issue also happen when you have multiple layouts configured.

I retried with:
```
KeyboardLayouts = "fr","us"
```

Shutting down the screen changes the keyboard layout back to "us".

Why does it change ? I did not unplug the keyboard, I only shut the screen off.
If you absolutely have to change, why choose "us", when my configuration file lists "fr" first ?

I also tried with:
```
KeyboardLayouts = "fr","de"
```

And upon screen shutdown it also changes to "us", even if it is not in the list of requested layouts.

But anyways, I'll try your patch
